### PR TITLE
WT-6446 Rename *.i into *_inline.h files

### PR DESCRIPTION
**Pull request by kommiharibabu**
WiredTiger use of XXX.i as the file name for inlined functions is a little odd,
generally the XXX.i suffix is reserved for files that do not require pre-processing,
and in the case of MSVC, it's a Visual Studio Intermediate file that contains
the output of the preprocessor.

### CODE REVIEW

**Review by tetsuo-cpp — APPROVED** [review]
LGTM.

**Review by sueloverso — COMMENTED** [review]
I have a couple small comments. One I think needs a change.

File: dist/prototypes.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def output(p, f):
 def prototypes_extern():
     p = []
     for name in source_files():
-        if not fnmatch.fnmatch(name, '*.[ci]'):
+        if not fnmatch.fnmatch(name, '*.c') + fnmatch.fnmatch(name, '*_inline.h'):
```
[the remark this answers is not in the document]
Comment (kommiharibabu) [reply dist/prototypes.py:57]: Even I don't have much knowledge, but from the following link, it is almost similar like C.
https://www.programiz.com/python-programming/precedence-associativity

File: dist/dist.py [hunk 2]
```diff
@@ -18,13 +18,13 @@ def source_files():
 #       Return list of all WiredTiger C source file names.
 def all_c_files():
     file_re = re.compile(r'^\w')
-    for line in glob.iglob('../src/*/*.[ci]'):
+    for line in glob.iglob('../src/*/*.c'):
```
[the remark this answers is not in the document]
Comment (kommiharibabu) [reply dist/dist.py:21]: Fixed by searching the *_inline.h files also.

**Review by sueloverso — APPROVED** [review]
LGTM

**Pull request merged**